File: demo/java-demo/src/main/java/com/alibaba/demo/basic/model/omni/PodStatus.java
package com.alibaba.demo.basic.model.omni;

public enum PodStatus {

    /**
     * waiting
     */
    WAITING("waiting"),

    /**
     * running
     */
    RUNNING("running"),

    /**
     * terminated
     */
    TERMINATED("terminated");

    PodStatus(String status) {}

}

File: demo/kotlin-demo/src/main/kotlin/com/alibaba/demo/basic/model/omni/PodStatus.kt
package com.alibaba.demo.basic.model.omni

enum class PodStatus(status: String) {
    /**
     * waiting
     */
    WAITING("waiting"),

    /**
     * running
     */
    RUNNING("running"),

    /**
     * terminated
     */
    TERMINATED("terminated")
}
